packages/convex-component/example/src/index.css

```
@@ -0,0 +1,133 @@
+  --warn: oklch(0.81 0.13 85);
+  --warn-dim: oklch(0.81 0.13 85 / 0.13);
+  --danger: oklch(0.72 0.17 25);
+  --danger-dim: oklch(0.72 0.17 25 / 0.14);
+  --neutral: oklch(0.7 0.015 60);
+  --neutral-dim: oklch(0.7 0.015 60 / 0.13);
+  --pending: oklch(0.78 0.1 240);
+  --pending-dim: oklch(0.78 0.1 240 / 0.14);
+
+  /* Type */
+  --font-sans:
+    ui-sans-serif, system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue",
```

Stylelint (17.12.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)